### ISSUE TITLE
cc-button: add bigger CTA mode

### ISSUE THREAD

**Issue by hsablonniere** [CleverCloud/clever-components#224]
Useful for the choose a payment method.

**Comment by florian-sanders-cc** [comment]
This is interesting because I am not sure these should be buttons anyway.

Currently, when you click one of these, it change the whole page content to display the card info form. 
It is not opening a new area that you can close or whatever, so it should probably be links ?

Still, maybe the current behavior (bringing the user to another "page" where there is only one form) could be reconsidered and we could do it differently ? (a tabpanel maybe ?)

Anyway, this subject might not be about the `cc-button` in the end :smile:

**Comment by hsablonniere** [comment]
Would you use a link that looks like a link?

I really see "paying" as a "user action". If you already had a credit card it may work directly (I'm not sure).

**Comment by florian-sanders-cc** [comment]
> Would you use a link that looks like a link?

No, in the current state (but I don't really like the current state), I would design them (visually I mean) as bigger CTAs, but I would use links instead of buttons in the code for semantics.

> I really see "paying" as a "user action".

Well, to me, paying is either the whole process or the submit action after filling the form, but not selecting the payment method. This process, is made of different actions, and the debate about semantics (link vs button) is not about the process but more about actions part of this process.
This process could be split into multiple pages or multiple actions within a signle page (opening a modal window or a detail / summary, filling a form, etc.).
This is mostly a design choice with pros and cons.

Here, these elements bring us to a "screen" / "page" (yes, line is blurry with SPAs and software), containing only one form, specific to the type of credit card you have clicked.

What makes me say it's a "screen" / "page" is that the whole content I was browsing when clicking one of these buttons is gone, I cannot access it anymore.

Let's say you have clicked the wrong card type (pretend you are absent minded like me :stuck_out_tongue: ), you have to go back to the previous "page" / "screen" in order to access the form you want. 
You can do this by using browser "previous page" feature (however, this does not always mean that one should use a link, as seen with Slack and how they handle modal / non-modal content).

> If you already had a credit card it may work directly (I'm not sure).

If so, I would imagine that the UI does not display 3 buttons to select the payment method, only one (which should probably be a button, depending on what it actually does).

Although we are debating whether it should be a button or a link, I think the actual subject might be to rethink (when we have time, cause clearly this is not a serious issue, it does the job as it is) the process ?
I would expect to select my payment method in the same screen as the one containing the form (what I mean is you usually have radio inputs / dropdown, then you fill the form below), except for 3rd party payments like Paypal in which case it might be on another page (which might very well be a paypal page or a page from your website with an iframe ?).
Right now the process splits these actions into 2 different parts, which feels unusual (but again, it's ok, just not as smooth ?)
